frontend/src/components/common/comments-list/comments-list.tsx

```diff
@@ -10,6 +11,7 @@ type Props = {
   user?: User | null;
   onTimeClick?: (payload: number) => void;
   onCommentDelete?: (commentId: number) => void;
+  toggleCommentLike?: (commentId: number) => void;
```

Comment:
```suggestion
  toggleCommentLike?: (commentId: number) => void;
```
check **A9** criteria